src/clients/sidecar-openapi-specs/sidecar.openapi.yaml

```diff
@@ -8,28 +8,28 @@ info:
     name: Confluent for VS Code Support
     url: https://confluent.io/contact
     email: vscode@confluent.io
-  version: 0.194.0
+  version: 0.196.0
 servers:
-  - url: http://127.0.0.1:26636
-    description: Auto generated value
+- url: http://127.0.0.1:26636
+  description: Auto generated value
```

Comment:
I'm not sure why `main`'s spec got some extra formatting, but wasn't the case for https://github.com/confluentinc/vscode/pull/1673/files#diff-cece9f10a62a57f71ad0f6027b4fd22cfc05e6fc68e96b0a4bbc2d3681eae498R12-R14